vibration/notifications.html

```diff
@@ -0,0 +1,27 @@
+---
+feature_name: NotificationOptions.vibrate
+chrome_version: 45
+feature_id: 5727996321202176
+---
+
+<p>When displaying a system notification your device may vibrate to alert the user. You can specify the vibration pattern that should be used in the same way as for the <a href="./index.html">Vibration API</a>.</p>
+
+<p>When creating the notification, just set the 'vibrate' property of the notification options to be a Vibration API pattern.</p>
+
+{% capture js %}
+var registrationPromise = navigator.serviceWorker.register('sw.js');
```

Comment:
Slight reworking:

    navigator.serviceWorker.register('sw.js');

    function showNotification() {
      Notification.requestPermission(function(result) {
        if (result === 'granted') {
          navigator.serviceWorker.ready.then(function(registration) {
            registration.showNotification('Vibration Sample', {
              body: 'Buzz! Buzz!',
              icon: '../images/touch/chrome-touch-icon-192x192.png',
              vibrate: [200, 100, 200, 100, 200, 100, 200],
              tag: 'vibration-sample'
            });
          });
        }
      });
    }

The `navigator.serviceWorker.ready` promise will only resolve when there's an active SW, which more closely matches the intent. And this checks the result of the `requestPermission()` call.